### PR TITLE
More moving Tags to common (3/5)

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/pathfinders/BotanicalExperience.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import {CardRequirements} from '../CardRequirements';
+import {all} from '../Options';
+import {Tags} from '../../common/cards/Tags';
+import {Size} from '../render/Size';
+import {ISpace} from '../../boards/ISpace';
+import {Board} from '../../boards/Board';
+import {ResourceType} from '../../common/ResourceType';
+import {IResourceCard} from '../ICard';
+
+export class BotanicalExperience extends Card implements IProjectCard, IResourceCard {
```
Comment (bafolts) [inline src/cards/pathfinders/BotanicalExperience.ts:16]: Woah? This seems out of place.

**Review by bafolts — APPROVED** [review]
I would remove the one card this seems to be adding from this commit prior to merging.

File: src/cards/pathfinders/BotanicalExperience.ts (hunk 1 above)
Comment (kberg) [reply src/cards/pathfinders/BotanicalExperience.ts:16]: Oh! Look at that, there's also a merge conflict.

Comment (kberg) [reply src/cards/pathfinders/BotanicalExperience.ts]: I'm sure resolving the merge conflict takes care of everything.

Comment (kberg) [reply src/cards/pathfinders/BotanicalExperience.ts]: Yep, taht fixed it. Thanks.

**Pull request merged**